bindings/Sofa/package/__init__.py

```diff
@@ -188,6 +189,12 @@ def __init__(self, *args, **kwargs):
         self.setName(str(self.__class__.__name__))
         self.addData("prefabname", value=type(self).__name__, type="string", group="Infos", help="Name of the prefab")
         self.addData("docstring", value=self.__doc__, type="string", group="Infos", help="Name of the prefab")
+        # args[0] should be the parent node. it has to be added NOW (before creating prefabParams) so that the links can be resolved before calling doReInit()
```

Comment:
It is probably the six's time I switch from with or without this "node" parameter.